api/python/provisioner/ui/README.md

```diff
@@ -0,0 +1,10 @@
+# Provisioner TUI
+
+This is northbound interface TUI for CORTX components that provides terminal UI
```

Comment:
Restructure required:
TUI (Terminal User Interface) provides user with a tool, which enables interaction with node configuration commands. This abstracts the need for user executing commands on shell terminal with necessary parameters; thus minimizing the possibilities of user errors and promoting usability.

Reply:
comment is addressed